### PR TITLE
Update local deploy

### PR DESCRIPTION
**Pull request by mikenairn**
Adds now required missing GLBC_KCP_CONTEXT=system:admin environment variable to local deployment configuration.

```
kubectl config use-context kind-kcp-cluster-glbc-control
make deploy
kubectl logs -f deployments/kcp-glbc-controller-manager -n kcp-glbc
```

Required since https://github.com/Kuadrant/kcp-glbc/pull/56

### CODE REVIEW

File: config/deploy/local/controller-config.env.template [hunk 1]
```diff
@@ -5,3 +5,4 @@ GLBC_ENABLE_CUSTOM_HOSTS=false
 GLBC_DOMAIN=dev.hcpapps.net
 GLBC_DNS_PROVIDER=aws
 AWS_DNS_PUBLIC_ZONE_ID=Z08652651232L9P84LRSB
+GLBC_KUBE_CONTEXT=system:admin
```
Comment (astefanutti) [inline config/deploy/local/controller-config.env.template:8]: nit: could this be renamed to `GLBC_KCP_CONTEXT`? There is already the `glbc-kubeconfig` option that is for K8S, that may be confused with `GLBC_KUBE_CONTEXT`, while the later is for KCP.

**Review by astefanutti — APPROVED** [review]

File: config/deploy/local/controller-config.env.template (hunk 1 above)
Comment (mikenairn) [reply config/deploy/local/controller-config.env.template:8]: true, i think the naming of the cli args probably wants looked as well as it's currently --context, but as you say we have two kubeconfigs, so what ones it for.

Comment (astefanutti) [reply config/deploy/local/controller-config.env.template:8]: Right, we might also want to rename the CLI options to something like `--kcpconfig`, `--kcp-context`, and `--kubeconfig`, `--kube-context`.

Comment (mikenairn) [reply config/deploy/local/controller-config.env.template]: Renamed the env var for now, will come back to what the CLI options might want to be.

**Pull request merged**